config20231115t1006.yml

```diff
@@ -0,0 +1,249 @@
+# MCMC Paramters
```

Comment:
I think it would be a good idea to add the mcmc parameters to the parameters.yml (and then if the mcmc isn't called those values just won't be used)

Reply:
Yes, good idea!